packages/e2e-tests/specs/experiments/blocks/comments-query.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activateTheme,
+	createNewPost,
+	insertBlock,
+	pressKeyTimes,
+	publishPost,
+	setOption,
+	trashAllComments,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Comment Query Loop', () => {
+	beforeAll( async () => {
+		await activateTheme( 'emptytheme' );
+		await setOption( 'page_comments', true, 'options-discussion.php' );
+		await setOption( 'comments_per_page', '1', 'options-discussion.php' );
```

Comment:
Would it make sense to consider returning the previous values after all (to use them in `afterAll` to reset those settings)? It might make the util a bit more useful in cases where we don't know (or don't want to look up) the default value. (Furthermore, it'd make our e2e tests a bit more resilient against changes of defaults.)